### DISCUSSION TITLE
How can i change the size of text-field?

### DISCUSSION THREAD

**Discussion by van23qf** [material-components/material-web#5144] [Q&A]
Is there any attr to change the size of text-field?

**Comment by asyncliz** [comment]
What size are you trying to change?

1. The width/height
  Use CSS `width` and `--md-outlined-text-field-container-height` (or `*-filled-*`)
1. The width in characters ([`size` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/size))
  Not currently implemented, but accepting PRs to add it!
3. The size of input/label text
  Use CSS `--md-outlined-text-field-input-text-size`, along with `*-input-text-line-height` or `*-label-text-*` for the label.
4. Something else?
